healthtools/scrapers/base_scraper.py

```diff
@@ -82,36 +85,37 @@ def scrape_site(self):
         '''
         This functions scrapes the entire website by calling each page.
         '''
-        self.set_site_pages_no()
+        self.site_pages_no = self.set_site_pages_no()
```

Comment:
@DavidLemayian Actually i made set_site_pages return the value of the pages so that we set it here. But considering the DRY principles, the initial implementation was better